.travis.yml

```diff
@@ -20,3 +20,18 @@ before_script:
   # http://docs.travis-ci.com/user/database-setup/#MySQL
   - sudo /etc/init.d/postgresql stop
    # - sudo /etc/init.d/mysql stop
+
+cache: pip
+
+deploy:
+    provider: pypi
+    user: misha.gavela
+    distributions: bdist_wheel
```

Comment:
sdist is a must, along with a wheel + it seems you forgot to make it universal (via `setup.cfg`)